kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/backstack/BackStackEffect.kt

```diff
@@ -1,15 +1,15 @@
-package com.squareup.viewregistry.backstack
+package com.squareup.workflow.ui.backstack
 
 import android.support.transition.Scene
 import android.view.View
 import android.view.ViewGroup
-import com.squareup.viewregistry.BackStackScreen
-import com.squareup.viewregistry.BackStackScreen.Key
-import com.squareup.viewregistry.ViewBinding
-import com.squareup.viewregistry.ViewRegistry
-import com.squareup.viewregistry.backstack.ViewStateStack.Direction
-import com.squareup.viewregistry.buildScene
-import com.squareup.viewregistry.buildView
+import com.squareup.workflow.ui.BackStackScreen
+import com.squareup.workflow.ui.BackStackScreen.Key
+import com.squareup.workflow.ui.ViewBinding
+import com.squareup.workflow.ui.ViewRegistry
+import com.squareup.workflow.ui.backstack.ViewStateStack.Direction
+import com.squareup.workflow.ui.buildScene
```

Comment:
Is `com.squareup.workflow.ui.android.*` too long a package name? It would be nice if the package names corresponded to the module names.

Reply:
Not so much that it's too long as that it's kind of an orthogonal concern, kind of redundant. Is that extra package name really useful to the android developer, or would it be cognitive clutter?

I did the same thing in the sample code -- the coordinators in the android module are in the same packages as the related screen objects in the common module.

Reply:
I'll merge this as-is, and follow up if you feel strongly.

Reply:
Yea, I don't feel strongly. And if you're writing a UI for non-android, you're not going to depend on both android and javafx (e.g.) artifacts anyway.